.circleci/rebase-pr.js

```diff
@@ -181,19 +185,18 @@ function getRefAndShas(sha, owner, name) {
   return {remote: remoteName, ref, latestSha, sha};
 }
 
-
 /** Gets the refs and shas for the base and target of the current environment. */
 function getRefsAndShasForChange() {
   const base = getRefAndShas(
     process.env['CIRCLE_GIT_BASE_REVISION'],
     process.env['CIRCLE_PROJECT_USERNAME'],
-    process.env['CIRCLE_PROJECT_REPONAME']
+    process.env['CIRCLE_PROJECT_REPONAME'],
```

Comment:
Are we using trailing commas? I'm actually in favor of them, but I think I remember getting nits about them from other members of our team

Reply:
I'm surprised if someone complained about trailing comma, but maybe there was. 

Trailing commas make a lot of sense IMO because it reduces the diff if I add new elements/properties, or if some are removed.

Reply:
My ruling is that we should be using trailing commas in almost every situation possible. Only exception is if everything fits on one line.